tox.ini

```diff
@@ -2,11 +2,11 @@
 # envlist is the list of envs that are tested when `tox` is run without any option
 envlist = first_startup, py27-lint, py27-lint-docstring-include-list, py27-lint-imports-include-list, py27-unit, py34-lint, qunit, validate-test-tools
 skipsdist = True
+skip_install = True
```

Comment:
I noticed that this change seems to have no effect on the Travis job logs, I think because it is redundant since we have `skipsdist = True`, see https://tox.readthedocs.io/en/latest/example/general.html?highlight=skipsdist#avoiding-expensive-sdist